server/src/main/proto/spine/server/aggregate/aggregate.proto

```diff
@@ -61,6 +71,8 @@ message Snapshot {
 //
 message AggregateEventRecord {
 
+    AggregateEventRecordId id = 4;
```

Comment:
Is it `(required)`? Assuming, this is not a state of an entity, the entity-related convention for identifiers does not apply here.